filebeat/module/elasticsearch/audit/manifest.yml

```diff
@@ -5,12 +5,15 @@ var:
     default:
       - /var/log/elasticsearch/*_access.log
       - /var/log/elasticsearch/*_audit.log
+      - /var/log/elasticsearch/*_audit.json
     os.darwin:
       - /usr/local/var/lib/elasticsearch/*_access.log
       - /usr/local/var/lib/elasticsearch/*_audit.log
+      - /usr/local/var/lib/elasticsearch/*_audit.json
     os.windows:
       - c:/ProgramData/Elastic/Elasticsearch/logs/*_access.log
       - c:/ProgramData/Elastic/Elasticsearch/logs/*_audit.log
+      - c:/ProgramData/Elastic/Elasticsearch/logs/*_audit.json
```

Comment:
I only added the new paths — as opposed to replacing the old `*_audit.log` ones — to allow for maximum cross-version compatibility between Filebeat and Elasticsearch.